spec/presenters/content_type_resolver_spec.rb

```diff
@@ -0,0 +1,97 @@
+require "rails_helper"
+
+RSpec.describe Presenters::ContentTypeResolver do
+  subject { described_class.new("html") }
+
+  it "inlines content of the specified content type" do
```

Comment:
Could you add a test of what happens if the `html` version isn't present?  ie, if it gets
`body: [ { content_type: "text", content: "some content" } ]`

I'm not sure if it should return `"body": null`, or omit `body` entirely.  It definitely shouldn't fall back to any other representation (because then downstream apps won't know what content type the data they get is).